tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "noEmit": true,
     "declaration": false,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
Needed to get around this issue: https://github.com/webdriverio/webdriverio/issues/10280

We do the same in other packages with browser tests too: https://github.com/cloudscape-design/board-components/blob/2a120411bf632a44ffc6256538d0b442be0a03ab/tsconfig.json#L17